pkg/daemon/daemon.go

```diff
@@ -578,18 +582,25 @@ func (dn *Daemon) nodeHasAnnotation(annoKey string, value string) bool {
 	return false
 }
 
+// isNodeDraining: check if the node is draining
+// both Draining and MCP paused labels will return true
 func (dn *Daemon) isNodeDraining() bool {
 	anno, ok := dn.node.Annotations[annoKey]
 	if !ok {
 		return false
 	}
 
-	if dn.openshiftContext.IsOpenshiftCluster() && !dn.openshiftContext.IsHypershift() {
```

Comment:
TBH, i think thats the only change needed to solve the race cond issue.

Reply:
if we do that we will return the previews race condition from here [https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/421](https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/421/files#diff-a53b7b593d3d778e62eaeeafa40088656f9212bfa2c2b7991df15fa78e60b0f0R587)

Reply:
ack